src/plugins/intel_cpu/tests/functional/subgraph_tests/src/concat_multiple_query_sdp.cpp

```diff
@@ -0,0 +1,296 @@
+ *               Concat         Transpose              Concat
+ *                / \               |                 /     \
+ *               /   \              |                /       \
+ *              /   MultiQuery      |           MultiQuery    \
+ *             /      \             |              /           \
+ *            /     Transpose       |          Transpose        \
+ *           /         \            |            /               \
+ *          Assign      ScaledDotProductAttention              Assign
+ *                                  |
+ *                               Tranpose
+ *                                  |
+ *                                Reshape
+ *                                  |
+ *                                 Add
+ *                                  |
+ *                                Result
+ */
+
+class ConcatMultiQuerySDPTest : public testing::WithParamInterface<ConcatMultiQuerySDPParams>,
+                                virtual public ov::test::SubgraphBaseTest,
+                                public CPUTestsBase {
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<ConcatMultiQuerySDPParams>& obj) {
+        ElementType inType;
+        InputShapeAndTransposeOrder inputShapeAndOrders;
+        bool hasShapeof;
+        std::tie(inType, inputShapeAndOrders, hasShapeof) = obj.param;
+        std::ostringstream result;
+        std::vector<InputShape>& inputShapes = inputShapeAndOrders.first;
+        std::vector<size_t>& transposeOrder = inputShapeAndOrders.second;
+        result << "IS=";
+        for (const auto& shape : inputShapes) {
+            result << ov::test::utils::partialShape2str({shape.first}) << "_";
+        }
+        result << "TS=";
+        for (const auto& shape : inputShapes) {
+            result << "(";
+            if (!shape.second.empty()) {
+                for (const auto& itr : shape.second) {
+                    result << ov::test::utils::vec2str(itr);
+                }
+            }
+            result << ")_";
+        }
+        result << "Prc=" << inType << "_";
+        result << "HasShapeOf=" << hasShapeof;
+        result << "TransposeOrder=";
+        result << "(";
+        for (const auto& itr : transposeOrder) {
+            result << itr << ",";
+        }
+        result << ")";
+
+        return result.str();
+    }
+
+    void SetUp() override {
+        InputShapeAndTransposeOrder inputShapeAndOrders;
+        bool hasShapeOf;
+        ElementType inType;
+        std::tie(inType, inputShapeAndOrders, hasShapeOf) = this->GetParam();
+        std::vector<InputShape>& inputShapes = inputShapeAndOrders.first;
+        std::vector<size_t>& transposeOrder = inputShapeAndOrders.second;
+        targetDevice = ov::test::utils::DEVICE_CPU;
+        rel_threshold = 1e-2f;
+        if (inType == ElementType::bf16) {
+            configuration.insert({"ENFORCE_BF16", "YES"});
```

Comment:
Please use `ov::hint::inference_precision(ov::element::bf16)` instead.
See https://github.com/openvinotoolkit/openvino/pull/21634

Reply:
Apply inference_precision for configuration, thanks!